### DISCUSSION TITLE
How to trim database?

### DISCUSSION THREAD

**Discussion by ForsakenRei** [aceberg/WatchYourLAN#146] [Q&A] [answered]
I know there is an option to trim history, but is there an option to trim database? e.g. a device hasn't been online for 1 month and then it will be removed from the database.
My use case is I have some VMs/LXCs for testing but it seems their history will always be there unless I drop the whole db?

**Comment by aceberg** [comment, marked as answer]
You can remove devices manually by clicking on their MAC and then `Delete host`.
I will consider adding a Trim option, though.

**Reply by ForsakenRei** [reply]
Ah thank you, didn't realized there is another page after click the MAC address. An option like forgot client after unseen for how long would be great.
